generated/nidaqmx/constants.py

```diff
@@ -822,10 +841,10 @@ class TaskState(Enum):
 
 
 class TemperatureUnits(Enum):
-    DEG_C = 10143  #: Degrees Celsius.
-    DEG_F = 10144  #: Degrees Fahrenheit.
-    K = 10325  #: Kelvins.
-    DEG_R = 10145  #: Degrees Rankine.
+    DEG_C = 10143  #: Deg C
+    DEG_F = 10144  #: Deg F
+    KELVINS = 10325  #: Kelvins
```

Comment:
Both and and new are wildly inconsistent. Maybe this should be DEG_K?

Reply:
`TemperatureUnits.K` is used in nidaqm-python repo. Thus have retained the expected enum value name (`K`)